yb-voyager/cmd/importSchema.go

```diff
@@ -81,7 +82,18 @@ func importSchema() {
 	}
 	objectList = applySchemaObjectFilterFlags(objectList)
 	log.Infof("List of schema objects to import: %v", objectList)
-	importSchemaInternal(&target, exportDir, objectList, nil)
+
```

Comment:
Add a comment stating: Import `ALTER TABLE` statements from the sequences.sql only after importing everything else.